.pre-commit-config.yaml

```diff
@@ -1,6 +1,6 @@
 repos:
     - repo: https://github.com/psf/black
-      rev: 19.10b0
+      rev: 20.8b1
       hooks:
         - id: black
-          language_version: python3.7
+          language_version: python3.8

```

Comment:
I think that line may be omitted. It is the first time I see it in any of our projects.